dataOrganization.md

```diff
@@ -0,0 +1,62 @@
+# Firestore Data Organization
+## Users
+### Collections
+* [Feeders](#feeders)
+* [Notifications](#notifications)
+### Document Fields
+```ts
+{
+  active: boolean,
+  appIdentifier: string,
+  email: string,
+  firstName: string,
+  id: string,
+  lastName: string,
+  lastOnlineTimestamp: timestamp,
+  phoneNumber: string,
+  profilePictureURL: string
+} 
+```
+## Feeders
+### Collections
+* [Pets](#pets)
+### Document Fields
+```ts
+{
+  id: string,
+  modelNumber: string,
+  name: string
+}
+```
+## Notifications
+### Document Fields
+```ts
+{
+  id: string,
+  message: string,
+  time: timestamp
+}
+```
+## Pets
+### Collections
+* [Schedule](#schedule)
```

Comment:
Should we simply associate food dispensing with pets, or does it need its own collection as stated here?

Is a schedule just how often food is dispensed? It doesn't change when pets are allowed to come to the feeder correct?

Reply:
We could associate the food dispensing with pets.

The reasoning for making it its own collections here was to allow for multiple schedule documents to exist on one pet without dealing with multidimensional arrays. This would allow the user to quickly swap between them if necessary. Saving the schedules as their own documents also allows for them to be moved to another pet's schedule collection by transporting the document rather than rewriting arrays.